install/kubernetes/helm/subcharts/mixer/values.yaml

```diff
@@ -18,11 +18,16 @@ telemetry:
   replicaCount: 1
   autoscaleEnabled: true
   autoscaleMin: 1
-  autoscaleMax: 5
+  autoscaleMax: 15
```

Comment:
any reason why we need to set this to 15 as the default?

Reply:
yeah, 15 assumes a very large cluster, where the new instances can spread out to different worker nodes.